diem-move/vm-genesis/src/lib.rs

```diff
@@ -258,10 +281,65 @@ pub fn encode_recovery_genesis_changeset(
     let (write_set, events) = convert_changeset_and_events(changeset1, events1).unwrap();
 
     assert!(!write_set.iter().any(|(_, op)| op.is_deletion()));
+
     verify_genesis_write_set(&events);
+    OLProgress::complete("Genesis transaction verified");
+
     Ok(ChangeSet::new(write_set, events))
 }
 
+/// fuction to iterate through a list of LegacyRecovery and recover the user accounts by calling a GenesisMigration.move in the VM. (as opposed to crafting writesets individually which could be fallible).
+
+fn migrate_end_users(session: &mut Session<StateViewCache<GenesisStateView>>, legacy_data: &[LegacyRecovery]) -> Result<u64, anyhow::Error>{
+
+  let filtered_data: Vec<&LegacyRecovery>= legacy_data
+  .iter()
+  .filter(|d| {
+        d.account.is_some() &&
+        d.account != Some(AccountAddress::ZERO)
+    })
+    .collect();
+
+
+
+    let mut total_balance_restored = 0u64;
+    for user in filtered_data.iter()
+    .progress_with_style(OLProgress::bar())
+    .with_message("Migrating user data") {      
+
+        let args = vec![
```

Comment:
Indentation issue and commented code without explexplanation

Reply:
cleaned up